### ISSUE TITLE
"unable to open database file"

### ISSUE THREAD

**Issue by patrickkwang** [ranking-agent/kp-registry#19]
> I'm also getting an error when running it locally:
>
> sqlite3.OperationalError: unable to open database file
>
> However, when running it through Strider, it does function properly. Not sure why this is happening, and it might be something funky going on with permissions on my computer.

-@uhbrar https://github.com/ranking-agent/kp-registry/pull/18#pullrequestreview-722658886

> On the sqlite error, I also saw that running locally via the main.sh, but not if I ran via docker-compose. So I think it has something to do with getting PYTHONPATH right or mounting the /data directory somehow.

-@cbizon https://github.com/ranking-agent/kp-registry/pull/18#issuecomment-892933450

**Comment by patrickkwang** [comment]
I just cloned a fresh copy of the repo, installed the requirements in a venv, and ran main.sh without any issues. Does @cbizon or @uhbrar have any more insight into this issue?

**Comment by cbizon** [comment]
I just did the same (but using conda instead of venv).

This was on master.

When I run main.sh it runs fine, but then executing refresh via the browser causes the error.

**Comment by uhbrar** [comment]
Yup, I'm on the same page as Chris. I cloned a fresh copy of the repo, recreated my venv directory, and reinstalled all requirements. I'm able to start is via main.sh, but if I try to actually do anything (get, refresh, anything at all), it causes the same sqlite error. @patrickkwang were you able to actually do anything, or did you just get it running?

As for other insights, I did run into this error 2 weeks ago, and I did something to fix it, but I just can't remember what that was, or how it got messed up again.

**Comment by patrickkwang** [comment]
Thanks for clarifying. It continues to work fine for me when I refresh and get KPs.
